src/linux/eventfd.rs

```diff
@@ -170,6 +171,12 @@ impl FromRawFd for EventFd {
     }
 }
 
+impl IntoRawFd for EventFd {
```

Comment:
Please, move this in a separate commit.

Reply:
Potentially could also get a CHANGELOG entry

Reply:
Not for each commit, but for each user visible change. It's fine if the link to the same PR.